models/models.js

```diff
@@ -63,5 +63,23 @@ exports.selectCommentsByArticleId = (article_id) => {
                     })
             }
         return result.rows;
-    })
+    });
+};
+
+exports.joinCommentToArticle = (article_id, username, body) => {
+    const sqlQuery = 'INSERT INTO comments (article_id, author, body) VALUES ($1, $2, $3) RETURNING *;'
+
+            return db.query("SELECT * FROM articles WHERE article_id = $1", [article_id]).then(({rows})=> {
```

Comment:
You don't need to query the articles table to check if the article_id exists before inserting a comment.